cpp/getting-started/how-to-run-aspose-slides-cpp-in-docker/_index.md

```diff
@@ -103,32 +103,35 @@ WORKDIR /slides-cpp/sample/
 CMD ./build_sample.sh
 ```
 
-## Build and run an image
+## Building and Running an Image
 
 1. [Install Docker](https://docs.docker.com/engine/install/) on a host system;
-2. Build an image. A terminal working directory should contain a file Dockerfile with the above content. 
+
+2. Build an image. 
+
+   A terminal working directory should contain a file Dockerfile with the content above. 
 
 ```
 docker build -t aspose-slides-ubuntu-16.04 .
 ```
 
 3. Download and unzip [Aspose.Slides for C++ YY.M Linux](https://downloads.aspose.com/slides/cpp);
-4. Share the folder with Aspose.Slides for C++ to Docker can use it(For Windows: Settings -> Resources -> File Sharing);
-5. Run the image as a container via one of following ways:
+4. Share the folder with Aspose.Slides for C++ to Docker can use it (For Windows: Settings -> Resources -> File Sharing);
```

Comment:
This line about sharing folders. You can see more details [here](https://token2shell.com/howto/docker/sharing-windows-folders-with-containers/).

Reply:
Go through the changes I made recently. If you agree with them, tell me "Yes" -- and I'll give my approval. Otherwise, if you don't agree with the changes, we can continue this conversation -- and you can make more changes.

Reply:
Yes. I agree with the changes.